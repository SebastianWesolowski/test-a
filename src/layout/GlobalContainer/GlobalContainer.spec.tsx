import renderer from "react-test-renderer";
import "@testing-library/jest-dom";

import { GlobalContainer } from ".";

describe("Components", () => {
  describe("GlobalContainer", () => {
    const component = renderer.create(<GlobalContainer />);
    const tree = component.toJSON();

    it("should render without errors", () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
