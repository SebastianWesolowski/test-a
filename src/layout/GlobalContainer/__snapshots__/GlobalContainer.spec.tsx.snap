// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components GlobalContainer should render without errors 1`] = `
<div
  className="flex min-h-screen flex-col"
/>
`;
